crates/types/src/line.rs

```diff
@@ -12,6 +12,17 @@ pub struct Line<const DIMENSION: usize>(pub Point<f32, DIMENSION>, pub Point<f32
 pub type Line2 = Line<2>;
 
 impl Line2 {
+    pub fn signed_angle(&self, other: Self) -> f32 {
+        (self.1 - self.0)
+            .normalize()
+            .dot(&(other.1 - other.0).normalize())
+            .acos()
+    }
```

Comment:
You are not using this function, right? I think you should remove it.

Reply:
Removed